versus-fe/src/App.js

```diff
@@ -19,24 +21,29 @@ import Drawer from './components/Drawer/Drawer';
 class App extends Component {
   constructor(props) {
     super(props);
+    console.log('user', this.props.user.user_id)
```

Comment:
Remove `console.logs` always. Your DevTools console is going to be a mess after some time.